string robot_name
string state
# can be: busy, ready
# TODO: eliminare se non utilizzato (robot pubblicano stato nella loro ns)